WalletWasabi/Crypto/StringCipher.cs

```diff
@@ -108,14 +108,13 @@ public static string Decrypt(string cipherText, string passPhrase)
 			return Encoding.UTF8.GetString(plainTextBytes, 0, decryptedByteCount);
 		}
 
-		private static AesManaged CreateAES()
+		private static Aes CreateAES()
```

Comment:
Just for the record: There was a discussion many years ago about this and the idea was to use the concrete instance just to make sure it cannot be changed by configuration. However, i think this is the right way.